securedrop_client/gui/widgets.py

```diff
@@ -1717,6 +1721,12 @@ class ReplyBoxWidget(QWidget):
         font-size: 18px;
         color: #9c9dbb;
     }
+    QTextEdit {
+        border: none;
```

Comment:
I agree that we need to get rid of this border around the text box and instead we need to refactor a bit to get the border here on a parent object:

<img width="801" alt="Screen Shot 2019-08-02 at 11 58 58 AM" src="https://user-images.githubusercontent.com/7832803/62383068-012e4780-b51d-11e9-990c-7778eda12339.png">